lib/processCss.js

```diff
@@ -97,6 +97,20 @@ var parserPlugin = postcss.plugin("css-loader-parser", function(options) {
 					item.nodes.forEach(processNode);
 					break;
 				case "item":
+					if (item.name.indexOf('progid:DXImageTransform.Microsoft.AlphaImageLoader') >= 0){
```

Comment:
Please add a comment (in brief) here, describing what the following code/case is doing